src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -560,6 +564,31 @@ private String zipFiles(String type, HttpContext httpContext,
 			return zipFileStatus.toString();
 		}
 
+		/**
+		 * return a RequestConfig Object with Timeout params set
+		 * @return
+		 */
+		private RequestConfig getRequestConfigWithTimeouts() {
+			// default time out to be 10 seconds
+			int timeout_miniseconds = 1000;
+			if (env.getProperty(Utils.ENV_PROPERTY_TIMEOUT_MINISECOND)!=null) {
```

Comment:
It would take very little to generalize this to a getIntegerProperty method in Utils.  That could be used to reduce duplication and ensure sanity checking of values.